styles/global.scss

```
@@ -21,3 +21,18 @@ input {
 p {
   margin: 0;
 }
+
+ul {
+  margin: 0;
+  padding: 0;
+}
```

SUGGESTION : You can resolve if you don't want to do this

```suggestion
p, ul {
  margin: 0;
}

ul {
  padding: 0;
}
```